README.md

```diff
@@ -4,23 +4,14 @@ Plugin for `Yazi` to preview all supported files via `Bat`.
 
 ## Preview
 
-Showing .csv files
+![preview](https://github.com/user-attachments/assets/67fe6e52-95c6-4768-a378-732e9c11cb84)
 
-![CSV via bat](./yazi-bat--csv.png)
-
-Previewing .md files
-
-![Markdown via bat](./yazi-bat--markdown.png)
-
-Previewing .rs files
-
-![Rust via bat](./yazi-bat--rust.png)
 
 ## Installation
 
 ```sh
 $> mkdir -p ~/.config/yazi/plugins
-$> git clone https://github.com/mgumz/yazi-plugin-bat.git ~/.config/yazi/plugins/bat.yazi
+$> git clone https://github.com/Pahasara/yazi-plugin-bat.git ~/.config/yazi/plugins/bat.yazi
```

Comment:
why would i merge this change of the url?

Reply:
Sorry about that! I unintentionally left the URL pointing to my fork. I'll fix it right away and update the PR.